packages/core/src/ChatMessageContainer/ChatMessageContainer.tsx

```diff
@@ -0,0 +1,53 @@
+/* istanbul ignore file */
+// todo: remove coverage ignore once storybook interaction test coverage counts
+
+import React from 'react'
+import styled from 'styled-components'
+import themeGet from '@styled-system/theme-get'
+import { Flex } from '../Flex'
+import { ChatMessage } from '../ChatMessage'
+
+const GapFlex = styled(Flex)`
```

Comment:
can you put the styled-components in a `ChatMessageContainer.styled.tsx`? that's the convention we're trying to adopt